api_tests/preprint_providers/views/test_preprint_provider_detail.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 from api.base.settings.defaults import API_BASE
-from api.preprint_providers.permissions import GroupHelper
+from api.providers.permissions import GroupHelper
```

Comment:
Hm, should the generalized tests be moved to `api_tests/provider/preprints/views/test_preprint_provider_detail.py`
That's an actual question... I think if we're deprecating might be better to make it clear where provider stuff lives now.